assets/dev/js/editor/elements/views/base.js

```diff
@@ -147,7 +148,8 @@ BaseElementView = BaseContainer.extend( {
 						icon: 'eicon-clone',
 						title: __( 'Duplicate', 'elementor' ),
 						shortcut: controlSign + '+D',
-						callback: () => $e.run( 'document/elements/duplicate', { container: this.getContainer() } ),
+						isEnabled: () => elementor.selection.isSameType(),
+						callback: () => $e.run( 'document/elements/duplicate', { containers: elementor.selection.getElements( this.getContainer() ) } ),
```

Comment:
Better `elementor.selection.getElements()` will know about current selected container without fallback if it's possible, if not, use `||` it's more readable approach.